GitHubCore.ps1

```diff
@@ -973,3 +973,36 @@ function Get-MediaAcceptHeader
 
     return ($acceptHeaders -join ',')
 }
+
+function Get-ContentMediaType
```

Comment:
I completely agree that we should have a method that does this.  But even I'm confused about when we use `Get-ContentMediaType` vs `Get-MediaAcceptHeader`, _especially_ when all uses for the methods are to assign a value to `AcceptHeader`.  Is there any way that these can be unified?  I'm ok with the `ValidateSet` being a superset given that the caller is doing its own `ValidateSet` validation.  Maybe there is an additional `AsJson` switch that the existing callers would pass in for it to add the `+json` to the end of the header.  I'm not quite sure, and I'm open to suggestions.  It's just that it's too confusing to leave it as-is, especially for the next person who comes along needing to use one and being confused which they should use.

Reply:
Yep i can modify the original method, i just didn't want to muck with the logic just in case but I totally agree that it's confusing to have both. I'll push an update.